collabnext/openalex/institutions.py

```diff
@@ -1,6 +1,34 @@
 from pyalex import Institution, Institutions
+import json
+import os
+import scripts.fetch_custom_institutions as fetch_custom_institutions
 
 
-def get_institutions() -> list[Institution]:
-    # Get 5 random institutions for now
-    return [Institutions().random() for _ in range(5)]
+def get_institutions(institutions_file_path: str = "observable/docs/data/institutions.json") -> list[Institution]:
```

Comment:
The path here doesn't work when running `invoke dev` because it runs from within the `observable` directory.

```suggestion
def get_institutions(institutions_file_path: str = "docs/data/institutions.json") -> list[Institution]:
```

Reply:
Updated in the new commit